### ISSUE TITLE
Is building with a more recent electron version (32) supported ? If not, is it being worked on ?

### ISSUE THREAD

**Issue by opsec** [ZUGFeRD/quba-viewer#75]
Is building with a more recent electron version (32) supported ? If not, is it being worked on ?

**Comment by jstaerk** [comment]
I dont know the answer to either of these questions but it's a I-guess-not on the second one. What would be the benefit? Do you volunteer?

**Comment by 5u623l20** [comment]
Recently I have ported quba-viewer on FreeBSD. It builds and works fine. Although I have been able to make it build and run on FreeBSD with Electron 32.1.2 my knowledge on the npm/electron ecosystem is not competitive enough to make a PR.

Within my limited knowledge simply bumping electron to 32.X.X should work. I can see that you have already bumped electron-builder to 24.13.3 on head which was the key problem while trying to build with 32.1.2.

In case you are interested here is the FreeBSD build scripts:
https://cgit.freebsd.org/ports/tree/finance/quba-viewer
